draft-ietf-httpbis-bcp56bis.md

```diff
@@ -239,7 +239,7 @@ their task cannot be completed).
 
 For example, an application can specify that it uses HTTP like this:
 
-    Foo Application uses HTTP {{!RFC7230}}. Implementations MUST support
+    Foo Application uses HTTP [RFC7230]. Implementations MUST support
```

Comment:
If the intent is to avoid creating a reference, then you want `\[RFC7230]`.  Functionally, this isn't a change at all.

BTW, if the intent is to blockquote this section, it should use `> Foo Application [...]`.